index.js

```diff
@@ -133,11 +133,13 @@ var createAssertion = function createAssertion(type, message, wrappers, block, m
 	if (descriptors.length === 0) {
 		throw new RangeError(inspect(type) + ' called with no wrappers defined');
 	}
-	var describeMsgs = [];
-	forEach(wrappers, function (wrapper) {
-		checkThis(wrapper);
-		describeMsgs.push(getThisDescription(wrapper));
```

Comment:
is the only change here that it's not pushing `getThisDescription(wrapper)` when the description is falsy? If so, I'd prefer a more minimal change here.